### PR TITLE
Add willowbark tea

### PR DESCRIPTION
**Pull request by Rail-Runner**
#### Summary
Content "Add willowbark tea"

#### Purpose of change

Add a historical way of using willow bark for pain treatment.
#### Describe the solution

Add JSON definition for willow bark tea and its recipe.
#### Describe alternatives you've considered

None.
#### Testing

Launched the game, made sure that no errors happen and willow bark tea gives painkiller effect if drunk (and if sober, too).
#### Additional context

Taken from MST Extra mod.

### CODE REVIEW

**Comment by Venera3** [comment]
Could you add this to the botany textbook as well? Adding it to some mid-to-high level chem/medical books might also make sense, as willowbark being a natural remedy is often cited as an example of natural products turning into modern drugs.

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -1008,6 +1008,18 @@
...
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_willowbark",
+    "copy-from": "tea",
+    "name": { "str_sp": "willowbark tea" },
+    "healthy": -1,
+    "fatigue_mod": 0,
+    "stim": 0,
+    "description": "A bitter tea made by boiling willow bark in water, used since ancient times as a mild painkiller.",
+    "fun": -2,
```
Comment (int-ua) [inline data/json/items/comestibles/drink.json:1020]: maybe remove the `EATEN_HOT` flag inherited from tea?

File: data/json/items/comestibles/drink.json [hunk 2]
```diff
@@ -1008,6 +1008,18 @@
...
+    "use_action": { "type": "consume_drug", "effects": [ { "id": "pkill1", "duration": 90 } ] }
```
Comment (actual-nh) [inline data/json/items/comestibles/drink.json:1021]: This should probably have a nausea (ideally, would be heartburn) side effect - why aspirin (acetylsalicylic acid) was invented was to try to tone down the negative stomach effects.

File: data/json/items/comestibles/drink.json [hunk 3]
```diff
@@ -1008,6 +1008,18 @@
     "flags": [ "EATEN_HOT" ],
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_willowbark",
+    "copy-from": "tea",
```
Comment (actual-nh) [inline data/json/items/comestibles/drink.json:1014]: From the tests/comestible_test.cpp down below - calories need to be set to 0, apparently?

File: data/json/items/comestibles/drink.json (hunk 2 above)
Comment (Maleclypse) [inline data/json/items/comestibles/drink.json:1021]:
```suggestion
    "use_action": { "type": "consume_drug", "effects": [ { "id": "pkill1", "duration": 90 },  { "id": "nausea", "duration": "5 m" } ] }
```

File: data/json/items/comestibles/drink.json (hunk 1 above)
Comment (Maleclypse) [reply data/json/items/comestibles/drink.json:1020]: ``
```suggestion
    "fun": -2,
    "delete": { "flags": [ "EATEN_HOT" ] },
```

File: data/json/items/comestibles/drink.json [hunk 4]
```diff
@@ -1008,6 +1008,18 @@
...
+    "name": { "str_sp": "willowbark tea" },
+    "healthy": -1,
+    "fatigue_mod": 0,
+    "stim": 0,
```
Comment (Maleclypse) [inline data/json/items/comestibles/drink.json:1018]:
```suggestion
    "stim": 0,
    "calories": 0,
```

**Review by Maleclypse — COMMENTED** [review]
I think we can make these changes and merge this.

**Review by Maleclypse — APPROVED** [review]
Assuming this passes tests it should be good now I think.

**Comment by actual-nh** [comment]
I currently have no clue why the PR validator is failing.

File: data/json/items/comestibles/drink.json [hunk 5]
```diff
@@ -1008,6 +1008,20 @@
...
+    "calories": 0,
+    "description": "A bitter tea made by boiling willow bark in water, used since ancient times as a mild painkiller.",
+    "fun": -2,
+    "delete": { "flags": [ "EATEN_HOT" ] },
+    "use_action": { "type": "consume_drug", "effects": [ { "id": "pkill1", "duration": 90 },  { "id": "nausea", "duration": "5 m" } ] }
```
Comment (actual-nh) [inline data/json/items/comestibles/drink.json:1023]:
```suggestion
    "use_action": { "type": "consume_drug", "effects": [ { "id": "pkill1", "duration": 90 }, { "id": "nausea", "duration": "5 m" } ] }
```

**Comment by Maleclypse** [comment]
> I currently have no clue why the PR validator is failing.

It has to be immediately under the bold Summary and not under the instructions that are hidden except when you are editing the PR.

**Pull request merged**